pages/about.html

```diff
@@ -0,0 +1,135 @@
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">News</a>
+          </li>
+        </ul>
+        <button class="btn btn-outline-success" type="submit">CC Campaign</button>
+      </div>
+    </div>
+  </nav>
+
+  <section class="first-section">
+    <div class="first-section-container">
+      <h3 class="second-heading">"Hello! Sharing world"</h3>
+      <h1 class="open-heading">SOFTWARE DEVELOPER GLOBAL SUMMIT 2021</h1>
+    </div>
+
+    <div class="about-paragraph">
+      <p class="about-us">
+        The Global Summit brings together the community of experts,academics and activist who comprise the Creative
+        Commons affialiate network in a different country every two years.<br>
+        This year,we hope to expand our invitation list including organization and individuals<span> who want to work
+          with us on shared projects that advance the cause of the Commons,free culture and open knowledge.<br>
+          So if you're active and engaged in the world of open content konwledge - free software advocates ,Wikipedian ,
+          Open Knownledge, galleries,libraries,museums,archives,goverments and foundations, lawyers , and avtivist - we
+          hope you'll consider joining us this year to build a stronger, more vibrant commons together</span>
+      </p>
+    </div>
+
+    <div class="contact">
+      <p class="p-contact">Please contact us per Email for any further questions about SD Global Summit 2021
+        !<br><strong>sdsummit2021@sdkosova.org</strong></p>
+    </div>
+  </section>
+
+  <section class="about-second-section">
+    <h3>Software Developers Gl<span class="span-2">ob</span>al Summit<span class="span-1">mi</span>t 2021 Logo</h3>
+    <p>The logo of SD Global Summit 2021 was decided throught the logo competition form 8.June to 7.July.
+    </p>
+    <img src="../images/logo1.png" alt="logo-image">
+    <hr>
+  </section>
+
+  <section class="events">
+    <h4>See the past S<span>D G</span>lobal Summits</h4>
+    <p>Take a look at the last two SD Global Summits which took place in Buenos Aires and in Warsaw.</p>
+    <div class="events-images">
+      <img src="../images/2017.png" alt="first-event" class="first-event">
+      <img src="../images/2019.png" alt="second-event" class="second-event">
+    </div>
+  </section>
+
+
+  <footer>
+    <div class="footer-flex">
+      <div class="footer-logo">
+        <a href="../index.html">SD Global Summit</a>
+      </div>
+      <p class="copyright">
+        2021 SD Global Summit
+        Some Rights Reserved
+      </p>
+    </div>
+  </footer>
+
+
+
+
+
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-U1DAWAznBHeqEIlVSCgzq+c9gqGAJn5c/t99JyeKa9xxaYpSvHU5awsuZVVFIhvj"
+    crossorigin="anonymous"></script>
+  <script src="https://kit.fontawesome.com/c70d972411.js" crossorigin="anonymous"></script>
+  <script src="/js/menu.js"></script>
```

Comment:
- Your file, it's missing. Kindly call the file correctly.
`` GET file:///js/menu.js net::ERR_FILE_NOT_FOUND ``